news/TEMPLATE.rst

```diff
@@ -6,6 +6,8 @@
 
 **Removed:** None
 
-**Fixed:** None
+**Fixed:**
+ - FIX: tests to run on windows OS by removing
+   removed directory paths
```

Comment:
I think you edited the template rather than the actual news file

Reply:
oops, sorry about that.